txpool/txpool_test.go

```diff
@@ -855,7 +856,139 @@ func TestSizeLimit(t *testing.T) {
 			err = pool.addImpl(tt.input.origin, genTx(t, &tt.input))
 			assert.Equal(t, tt.err, err)
 			assert.Equal(t, tt.len, pool.Length())
-			assert.Equal(t, tt.slots, pool.slots)
+			assert.Equal(t, tt.slots, pool.slotsOccupied())
+		})
+	}
+}
+
+func TestGaugeCheck(t *testing.T) {
+	type AddTx struct {
```

Comment:
There is a similar structure in the test above. We should strive to code reusability. Consider taking common things out of the test methods

Reply:
[Fixed](https://github.com/0xPolygon/polygon-sdk/pull/213/commits/ce80cf9d857d332b6c1258016cbccee67526cfbd)